src/config.rs

```diff
@@ -686,6 +686,10 @@ impl Config {
                 let value = value.unwrap_or("yes").into();
                 self.remove_make = validate(value, yes_no_ask)?;
             }
+            "SkipReview" => {
+                let value = value.unwrap_or("ask").into();
```

Comment:
Also the default forr ask if very weird.

setting `SkipReview` without a value should be always.

Also ask is the default if you don't put anything so there's no sense in it being a valid option.

Just `SkipReview = Always|Never`

Reply:
> It's awkard because it's inverted.

> setting SkipReview without a value should be always.

> Also ask is the default if you don't put anything so there's no sense in it being a valid option.
> Just SkipReview = Always|Never

I agree with these, should i go ahead and fix it?

> Well a diff is the differences between when you last saw it and now. So if there's nothing to show there's nothing to show. Even if set to always show diffs.

Yes, but then it feels somewhat misleading to call the option `always` as it's _not_ always. Maybe it should be called anything else? The documentation should atleast be updated to be clear about this.